### DISCUSSION TITLE
How to add a word wrap indicator to the terminal?

### DISCUSSION THREAD

**Discussion by vanarok** [termux/termux-app#3746] [Q&A] [answered]
![Screenshot_20231218-111246_Termux](https://github.com/termux/termux-app/assets/6105977/02037b24-a90f-4712-9cc0-70c3ce2b02a7)
That's what I mean:
![Screenshot_20231218-111324_Termux](https://github.com/termux/termux-app/assets/6105977/c4d705e2-db19-494d-b430-4b7104b15507)

**Comment by twaik** [comment, marked as answer]
Terminals do not have such a switch. All terminal applications must implement it on their own.
